app/views/listings/form/_currency_selector.haml

```diff
@@ -1,3 +1,2 @@
 .currency-selector
   = Money.new(1, @current_community.currency).symbol
-  = form.hidden_field :currency, :value => @current_community.currency
```

Comment:
Originally the currency was passed through the form as a hidden field. The commit that this PR reverts changed that to read the currency straight from the community model.